### DISCUSSION TITLE
"View" or "Show" page for resource

### DISCUSSION THREAD

**Discussion by msnegurski** [filamentphp/filament#146] [Ideas]
Right now the default page for resource is "edit" page, which is a form. Would be great to have "view" or "show" page, that just has all the information about resource that is not a form, but just a page with information.

**Comment by iantasker** [comment]
I would like to have the edit button off the view page. It would work with permission in a slightly nicer way.

**Reply by pqr** [reply]
There are two kind of CRUD admin panels:
1. List + Edit pages
2. List + View + Edit pages

I've build dozen projects with different admin panels and from my experience "List + View + Edit" gives better UX for end user. Fully support this topic!

**Reply by ryangjchandler** [reply]
We've spoken about the idea already. I personally think it should be an opt-in, since I actually prefer not having the View page. That's simple enough to do though since Filament just registers pages anyway.

**Reply by msnegurski** [reply]
@ryangjchandler at my current work more than 90% of cases to view a resource is when user asks something and support manager opens his profile in admin panel with some logs etc. to view what user did and explain the situation. And a small minority of cases when there is a need to edit something. Actually I usually disable editing by default and only allow it in rare cases or through actions.

Of course I am biased, but it's just my 5 cents about usecase. Nova handles this pretty well, so I thought it would be a useful feature for Filament to have.

**Reply by bernhardh** [reply]
Just to give my two cents to this topic: I also prefere to not have a view page. Especially the solution how laravel nova solved it is very bad from my perspective, since you have to switch from form to "show", if you like to edit the relations. I think, thats very bad usability and thats one of the things I get the most complains about from my customers who uses nova.

**Reply by ryangjchandler** [reply]
@bernhardh I think we're going to go ahead with adding "View" pages but it won't be a default, you'll have to opt-in.
